srcpkgs/cljstyle/template

```diff
@@ -0,0 +1,62 @@
+# Template file for 'cljstyle'
+pkgname=cljstyle
+version=0.16.626
+revision=1
+hostmakedepends="clojure mandrel"
+makedepends="zlib-devel"
+short_desc="Tool for formatting Clojure code"
```

Comment:
Why not "Clojure code formatter"?
In this way it is easy to search formatters: https://voidlinux.org/packages/?arch=x86_64&q=formatter